popde/density_estimate.py

```diff
@@ -1,3 +1,83 @@
+class SimpleKernelDensityEstimation:
+    """
+    Kernel Density Estimation (KDE) using Kernels
```

Comment:
no point to say that KDE uses kernels I think .. 
```suggestion
    Fit and evaluate general Kernel Density Estimation (KDE)
```

Reply:
fixed